workflows/alevin-quant/run-alevin.nf

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env nextflow
+nextflow.enable.dsl=2
+
+// run parameters
+params.ref_dir = 's3://nextflow-ccdl-data/reference/homo_sapiens/ensembl-100'
+params.index_dir = 'salmon_index'
+params.index_name = 'cdna_k31'
+params.annotation_dir = 'annotation'
+params.t2g = 'Homo_sapiens.ensembl.100.tx2gene.tsv'
+params.mitolist = 'Homo_sapiens.ensembl.100.mitogenes.txt'
+params.sample_dir = 's3://ccdl-scpca-data/raw/green_adam'
+params.sample_id = ['834', '905_3']
```

Comment:
Just for my understanding -- are these specific to your tests? How would these be specified in a regular run?

Reply:
Later, the workflow{} map{} function iterates over the params.sample_id list. Right? This represents two distinct samples from the sample data provider?

Reply:
That is correct.